test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/rest/RestClient.java

```diff
@@ -2,18 +2,26 @@
 
 import static javax.ws.rs.client.Entity.entity;
 import static javax.ws.rs.core.HttpHeaders.ACCEPT;
+import static uk.gov.justice.services.test.utils.core.rest.ResteasyClientBuilderFactory.clientBuilder;
 
-import javax.ws.rs.client.ClientBuilder;
 import javax.ws.rs.client.Entity;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.MultivaluedMap;
 import javax.ws.rs.core.Response;
 
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+
 /**
  * Helper class to send Post Commands and Get Queries.
  */
 public class RestClient {
 
+    private final ResteasyClientBuilder clientBuilder;
```

Comment:
Do you need to store the client builder as a member variable? Surely as it's created using a static method it could be created anew each time it's needed